kymatio/backend/tensorflow_backend.py

```diff
@@ -1,23 +1,5 @@
-import tensorflow as tf
 import numpy as np
-
-
-
-
-def complex_check(x):
-    if not _is_complex(x):
-        raise TypeError('The input should be complex.')
-
-def real_check(x):
-    if not _is_real(x):
-        raise TypeError('The input should be real.')
-
-def _is_complex(x):
-    return (x.dtype == np.complex64) or (x.dtype == np.complex128)
-
-
-def _is_real(x):
-    return (x.dtype == np.float32) or (x.dtype == np.float64)
+import tensorflow as tf
```

Comment:
Do we need to put the tf backend into a class? What inherits from it? Can't it just stay as is?

Reply:
Just for consistency.